src/services/data_cache/cache.h

```diff
@@ -15,50 +15,74 @@
  * Used for all getting/setting raw data points
  */
 class DataCache {
-private:
-    std::unordered_map<std::string, boost::variant<float, uint64_t>> cache;
-    std::vector<std::string> keys;
-
 public:
     DataCache();
     ~DataCache();
 
+    enum Key : int {
+        // system usage
+        SYS_DISK_AVAIL = 0,
+        SYS_DISK_USED = 1,
+        SYS_MEM_AVAIL = 2,
+        SYS_MEM_USED = 3,
+        SYS_CPU_USED = 4,
+        SYS_CPU_TEMP = 5,
+        SYS_GPU_USED = 6,
+        SYS_GPU_TEMP = 7,
+        // nmea 0183
+        GPS_FIX_STATUS = 8,
+        GPS_FIX_TYPE = 9,
+        GPS_FIX_QUALITY = 10,
+        GPS_FIX_ALTITUDE = 11,
+        GPS_FIX_LATITUDE = 12,
+        GPS_FIX_LONGITUDE = 13,
+        GPS_FIX_SPEED = 14,
+        GPS_FIX_TRAVELANGLE = 15,
+        GPS_FIX_TRACKINGSATELLITES = 16,
+        GPS_FIX_VISIBLESATELLITES = 17,
+        GPS_FIX_TIMESTAMP_RAWTIME = 18
+    };
```

Comment:
lmk if you have any suggestions for making this better. We need to have a key field on data points to implement getting a single series of data through the REST API. I used integers, which map to these constants. Using ints rather than the actual strings will allow us to rename the fields more easily if we need to.